server/questions.js

```diff
@@ -206,26 +183,26 @@ exports.checkAnswer = function(questionId, user, answer, callback) {
 
 		if (question.type === common.questionTypes.MATCHING.value && answer) {
 		    const ansLeftSide = answer[0];
-				const ansRightSide = answer[1];
-
-				if (ansLeftSide.length === question.leftSide.length) {
-					var checkIndexLeft;
-					var checkIndexRight;
-
-					for (i = 0; i < ansLeftSide.length; i++) {
-					    checkIndexLeft = question.leftSide.indexOf(ansLeftSide[i]);
-							checkIndexRight = question.rightSide.indexOf(ansRightSide[i]);
-							if (checkIndexLeft !== checkIndexRight) {
-							    value = false;
-							}
-					}
+			const ansRightSide = answer[1];
```

Comment:
hmmm this looks a little off, could you double check that its all aligned

Reply:
I have aligned it properly, github changes the formatting